connect/api/src/main/java/org/apache/kafka/connect/data/Values.java

```diff
@@ -177,7 +213,12 @@ public static Long convertToLong(Schema schema, Object value) throws DataExcepti
      * @throws DataException if the value could not be converted to a float
      */
     public static Float convertToFloat(Schema schema, Object value) throws DataException {
```

Comment:
A few of these `convertTo<>()` methods are not covered by unit tests. It's ok not to address this in this PR if you'd prefer as it's already huge.

Reply:
I added tests to get the methods themselves up to 100% coverage, but the overall class still is missing some coverage. Thanks for pointing this out, as there were certainly some pretty obvious cases that weren't tested.